semver.js

```diff
@@ -704,6 +704,48 @@ Comparator.prototype.test = function(version) {
   return cmp(version, this.operator, this.semver, this.loose);
 };
 
+Comparator.prototype.intersects = function(comp, loose, platform) {
+  if (!(comp instanceof Comparator)) {
+    throw new TypeError('a Comparator is required');
+  }
+
+  var rangeTmp;
+
+  if (this.operator === '') {
+    rangeTmp = new Range(comp.value, loose, platform);
+    return satisfies(this.value, rangeTmp, loose, platform);
+  } else if (comp.operator === '') {
+    rangeTmp = new Range(this.value, loose, platform);
+    return satisfies(comp.semver, rangeTmp, loose, platform);
+  }
+
+  var sameDirectionIncreasing = (this.operator === '>=' || this.operator === '>') && (comp.operator === '>=' || comp.operator === '>');
```

Comment:
These really long lines are very hard to read.

Can you please break this up into separate stanzas?  There should be no lines longer than 80 chars under any circumstances.  70 is an even better limit.